### DISCUSSION TITLE
Preview is crashed after deploy

### DISCUSSION THREAD

**Discussion by Prodigydev99** [vercel/community#1660] [Help]
### Summary

Good afternoon, we have a problem on our project, after making changes in the development process and then deploying to vercel via github (build was successful and without errors) when we try to view the preview we get to a page with an error, without any additional information, it just says 'Something's wrong here...
We found an error while loading this page.

Please refresh or go back to the home page.'
Everything was fine before, no critical changes that might have affected this, can you please help, thanks 

No errors in the build, just at one point the preview stopped running 
<img width="1240" alt="image" src="https://user-images.githubusercontent.com/126520421/222146233-c3fabc76-fb6e-4851-a96e-6f6d6e1adbe2.png">

### Additional information

Error message: 
'Something's wrong here...
We found an error while loading this page.
Please refresh or go back to the home page.'

<img width="1437" alt="image" src="https://user-images.githubusercontent.com/126520421/222146122-f0437bb5-6dbf-48f5-aebe-4ebf6ca96a78.png">

### Example

https://groundtruth-shopify-hydrogen-git-staging-groundtruth-shopify.vercel.app/

**Comment by amyegan** [comment]
Thanks for reaching out @Prodigydev99. That looks like a custom error message rather than something coming directly from Vercel. The deployment's [Function Logs](https://vercel.com/guides/where-are-vercel-logs#function-logs) might also give more information about what is going wrong. Do you get the same error if you redeploy an older preview deployment that is known to work?
